qutebrowser/config/configdata.yml

```diff
@@ -1171,6 +1171,27 @@ statusbar.position:
   default: bottom
   desc: Position of the status bar.
 
+statusbar.widgets:
+  default: '{url} [{scroll}] [{history}] [{tabs}]'
+  type:
+    name: FormatString
+    fields:
+      - url
+      - scroll
+      - scroll_raw
+      - history
+      - tabs
+    none_ok: true
+  desc: |
+    Format to use for the statusbar widgets
+    The following placeholders are defined:
+
+    * `{url}`: The current page URL.
```

Comment:
For consistency, could you drop the article? (see https://github.com/qutebrowser/qutebrowser/commit/56671e578707f05c8e580705ef4eab61c0069e93)

Reply:
Done